src/components/Calendar.tsx

```diff
@@ -107,33 +89,27 @@ export const Calendar: FC<CalendarProps> = ({
 						</div>
 
 						<div className="pt-1">
-							{tasks.water.map(plantName =>
-								renderRemainder(
-									plantName,
-									day,
-									'bg-blue-300',
-									'text-blue-900',
-									<WaterIcon className="w-5 h-5 fill-blue-900" />
-								)
-							)}
-							{tasks.repot.map(plantName =>
-								renderRemainder(
-									plantName,
-									day,
-									'bg-amber-200',
-									'text-yellow-700',
-									<PlantPotIcon className="w-5 h-5 fill-yellow-700" />
-								)
-							)}
-							{tasks.fertilize.map(plantName =>
-								renderRemainder(
-									plantName,
-									day,
-									'bg-red-300',
-									'text-red-700',
-									<FertilizeIcon className="w-5 h-5 fill-red-700" />
-								)
-							)}
+							{tasks.water.map(plantName => (
+								<TaskFlag
+									key={`${plantName}-${day}`}
```

Comment:
key must be unique -- if there is repot and watering and one day it can make errror